lab_2/main.py

```diff
@@ -54,6 +92,21 @@ def calculate_distance(unknown_text_vector: list, known_text_vector: list) -> fl
     :param known_text_vector: vector for known text
     """
     pass
+    if not isinstance(unknown_text_vector,list) or not isinstance(known_text_vector,list):
+        return None
+    for el_unknown_vector in unknown_text_vector:
+        if not isinstance(el_unknown_vector, (float, int)):
+            return None
+    for el_known_vector in known_text_vector:
+        if not isinstance(el_known_vector, (float, int)):
+            return None
+    distance = 0
+    for ind_and_value in enumerate(unknown_text_vector):
```

Comment:
`enumerate` is usually used as `for idx, element in enumerate(some_iterable)` so that you do not have to use indexes inside the loop.

Reply:
If I split ind_and_value into inx and element then there is an error in Code Style

Reply:
I'd like to see it, it's ok for now.